lib/src/klio/transforms/helpers.py

```diff
@@ -765,3 +766,20 @@ def expand(self, pcoll):
             )
             | lbl3 >> beam.io.WriteToPubSub(topic=self.upstream_topic)
         )
+
+
+# TODO: maybe expose a way for users to add messages to the pubsub message
+# manager as well? no idea if that'd be useful/needed
+class KlioAckInputMessage(beam.DoFn):
+    """Message acknowledgement DoFn.
+
+    Marks a KlioMessage as done after it is finished processing.
+
+    Used in conjunction with the ``klio_exec.runners.evaluators.
```

Comment:
Non-blocking. Might be worth adding that this only needs to be used with DirectGKERunner so someone doesnt use the Dataflow runner, wonder why the UI isnt acking messages as they expect thinking that they need this transform.